README.md

```diff
@@ -306,6 +306,87 @@ spec:
 ```
 <b>Note:</b> Ensure that `insecure_skip_tls_verify` is set to `false` so that TLS is used.
 
+### OLM Integration
+Create `oadp-operator-source.yaml` file like below in oadp-operator directory:
+```
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: oadp-operator
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: deshah
+  displayName: "OADP Operator"
+  publisher: "deshah@redhat.com"
+```
+Remove the deployed resources by following all the commands below in Cleanup Section.
+
+<b>Note:</b> All the commands should be run by being at root of oadp-operator directory.
+
+Run the following commands below:
+
+```
+oc create namespace oadp-operator
+oc project oadp-operator
+oc create secret generic <SECRET_NAME> --namespace oadp-operator --from-file cloud=<CREDENTIALS_FILE_PATH>
+oc create -f oadp-operator-source.yaml
+```
+- After running these commands, install OADP Operator from OperatorHub.
+
+In order to use OLM for OADP deployment , you need to change flag `olm_managed` in the `konveyor.openshift.io_v1alpha1_velero_cr.yaml` to true. The file is present in deploy/crds folder.
```

Comment:
When you install from OLM and use the web console to create the CR we will have `olm_managed: true` as the default. I would just call this out that the user shouldn't need to tweak it if they create the CR from the console.

Reply:
We don't use the web console to create CR, as we had discussed.